explainer.md

```diff
@@ -34,7 +34,7 @@ For example, we wish to mitigate
 attacks on:
 
 - Users' routers, as outlined in
-  [SOHO Pharming](https://www.team-cymru.com/ReadingRoom/Whitepapers/2013/TeamCymruSOHOPharming.pdf).
+  [SOHO Pharming](https://331.cybersec.fun/TeamCymruSOHOPharming.pdf).
```

Comment:
I believe we can replace this with `[SOHO-PHARMING]`, since it is linked in the bibliography. I missed this when fixing #49, thanks for catching it!

Reply:
Woops! This is because my suggestion makes no sense! Missed that this was the explainer...